dawn/src/driver-includes/cuda_verify.hpp

```diff
@@ -46,15 +46,13 @@ struct compute_error<RelErrTag> {
 
 namespace dawn {
 template <typename error_type>
-__global__ void compare_dense_full_kernel(const int edge_start_idx_c, const int edge_end_idx_c,
-                                          const int dense_size_edges, const int k_size,
-                                          const double* __restrict__ dsl,
-                                          const double* __restrict__ fortran,
-                                          double* __restrict__ error) {
+__global__ void compare_kernel(const int start_idx, const int end_idx, const int stride,
```

Comment:
```suggestion
__global__ void compare_dense_kernel(const int start_idx, const int end_idx, const int stride,
```
cannot be used for sparse (explained above)